ansible/roles/ewf-app-config/files/deployment-scripts/backend_deployment.yml

```diff
@@ -20,7 +20,6 @@
         - "EWF"
         - "COM"
       supportscripts_file: "supportscripts"
```

Comment:
You can move `supportscripts` to the `directories` list now as it will be moved to the same final location.